code/modules/mob/living/brain/brain_item.dm

```diff
@@ -113,24 +113,31 @@
 	if(istype(O, /obj/item/borg/apparatus/organ_storage))
 		return //Borg organ bags shouldn't be killing brains
 
-	if((organ_flags & ORGAN_FAILING) && O.is_drainable() && O.reagents.has_reagent(/datum/reagent/medicine/mannitol)) //attempt to heal the brain
+	var/healby = 2 //heals 2 damage per unit of mannitol
+	var/pour = "carefully pour"
+	var/onto = "onto"
+	if(istype(O, /obj/item/reagent_containers/syringe))
+		healby = 4 //heal 4 damage because PRECISCION
+		pour = "precisely inject"
+		onto = "into"
+	else if(istype(O, /obj/item/reagent_containers/dropper))
+		healby = 4
+
+	if(O.reagents?.has_reagent(/datum/reagent/medicine/mannitol)) //attempt to heal the brain
```

Comment:
I think, with this code as written, you'd be able to "pour" any mannitol containing item onto the brain, including slices of delicious brain cake, or a cigarette injected with mannitol.

Reply:
im having trouble coming up with a simple, elegant check that would differentiate between valid and invalid containers for this.
valid examples:
- beaker
- bottle
- syringe
- dropper 
- ice cup/flask/drink with mannitol pill in it
invalid examples:
- food items
- cigarettes
- pill

im thinking about going back to is_drainable, but with special case to allow droppers and syringes